tests/integration/allocator_accessibility.cpp

```diff
@@ -154,6 +185,8 @@ void run_access_test(umpire::Allocator* alloc, size_t size)
   if(umpire::is_accessible(umpire::Platform::undefined, *alloc)) {
     FAIL() << "An Undefined platform is not accessible." << std::endl;
   }
+  else
+    SUCCEED();
```

Comment:
Minor comment:

Usually, when I have an `if` or `else` block that calls a macro, I surround the block with `{}`.  This is because I cannot always be certain if the macro will expand to mulitple lines or not and whether the macro implementor will place the macro contents within `{}`.
```suggestion
    {
        SUCCEED();
    }
```
If you agree, feel free to change this and the other if/else blocks above.

Reply:
Nice! Good idea